src/app/screens/BookList/layout.js

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+import { FlatList, Text } from 'react-native';
+
+import Book from './components/Book';
+import styles from './styles';
+
+import WithLoading from '../../components/WithLoading';
+
+class BookList extends PureComponent {
+
+  renderItem = ({item}) =>  <Book name={item.title} author={item.author} imageSource={{uri: `https${item.image.url.substring(4)}?id=${item.id}`}} onPress={() => this.props.handlePressBook(item)}/>;
+
+  keyExtractor = (item) => `${item.id}`;
+
+  render() {
+    const { books, getBooksErrorMessage } = this.props;
+
+    const errorMessage = !getBooksErrorMessage ?  'No books available at library.' : getBooksErrorMessage;
+    const error = <Text style={styles.errorMessage}>{errorMessage}</Text>;
+    
+    return (
+      <FlatList 
+        data={books} 
+        renderItem={this.renderItem} 
+        keyExtractor={this.keyExtractor} 
+        contentContainerStyle={styles.container}
+        ListEmptyComponent={error}
```

Comment:
Put directly the component here: `ListEmptyComponent={<Text style={styles.errorMessage}>{getBooksErrorMessage ? getBooksErrorMessage : NO_AVAILABLE_BOOKS}</Text>}`